docs/can-ndjson-stream.md

```diff
@@ -0,0 +1,69 @@
+@module {function} can-ndjson-stream
+@parent can-ecosystem
+@package ../package.json
+
+@description Parses an ndjson stream into a stream of JavaScript objects.
+
+@signature `ndjsonStream(stream)`
+
+The `can-ndjson-stream` module converts a stream of ndjson to a [ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream) of JavaScript objects. In order to use this module, you must first use [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) with a service that sends an ndjson stream.
+
+```js
+  const ndjsonStream = require('can-ndjson-stream');
```

Comment:
I think it would be reasonable, but I think Justin preferred two examples. I could also come up with a different example for the second one, but it would still be very similar.